src/tutorials/jbearing2.c

```diff
@@ -493,9 +493,10 @@ PetscErrorCode CallPermonAndCompareResults(Tao tao, void *ctx)
   PetscReal      rtol,atol,dtol;
   PetscInt       maxit;
   PetscReal      gatol, grtol, gttol;
+  PetscReal      rhs_norm;
+  PetscReal      tao_diff_tol = 1e2*PETSC_SQRT_MACHINE_EPSILON;
```

Comment:
@jkruzik Please commit suggestion if ok:
```suggestion
  PetscReal      rhs_norm, tao_diff_tol;
```